Source/SwiftLintFramework/Helpers/XCTestHelpers.swift

```diff
@@ -0,0 +1,24 @@
+import SourceKittenFramework
+
+private let testFunctionNames: Set = [
+    "setUp()",
+    "setUpWithError()",
+    "tearDown()",
+    "tearDownWithError()"
+]
+
+private let testVariableNames: Set = [
+    "allTests"
+]
+
+enum XCTestHelpers {
+    static func isXCTestMember(kind: SwiftDeclarationKind, name: String) -> Bool {
+        if SwiftDeclarationKind.functionKinds.contains(kind) {
+            return name.hasPrefix("test") || testFunctionNames.contains(name)
+        } else if SwiftDeclarationKind.variableKinds.contains(kind) {
+            return testVariableNames.contains(name)
```

Comment:
Same here, seems like we should only be checking for a static var called `allTests`, otherwise this would be incorrectly counted as a test member:

```swift
final class MyTests: XCTestCase {
  let allTests: [TestInput] { ... }
}
```